codonPython/tests/SQL_connections_test.py

```diff
@@ -0,0 +1,16 @@
+'''test script for SQL_connections
+- test the connections can run a dummy script (SELECT 1 as [Code], 'test' as [Name])'''
+from codonPython import SQL_connections
```

Comment:
This import isn't needed

Reply:
removed "from codonPython import SQL_connections" as it was unused